pkg/parser/parser.y

```diff
@@ -3959,6 +3959,13 @@ BuiltinFunction:
 			Args:   $3.([]ast.ExprNode),
 		}
 	}
+|	"REPLACE" '(' ExpressionListOpt ')'
```

Comment:
Can we remove this derivation and use the above derivation (`identifier '(' ExpressionList ')'`)?

Reply:
Done.